examples/application/evaluate_gaming_traffic.cc

```diff
@@ -56,49 +49,51 @@ GeneratedPacketsStats (Ptr<OutputStreamWrapper> stream, Ptr<const Packet> packet
   lastPacketTime = currentPacketTime;
   *stream->GetStream () << packet->GetSize ()  << ","
                         << packetsInterArrivalTime.GetSeconds () << std::endl;
-}
 
-void
-AccumulateDelay (Ptr<const Packet> packet, const Address & addr)
-{
-  TimestampTag timestamp;
-  if (packet->FindFirstMatchingByteTag (timestamp))
-    {
-      Time packetTime = timestamp.GetTimestamp ();
-      delayAccumulator += Simulator::Now () - packetTime;
-    }
 }
 
 void
-CalculateThroughput (Ptr<OutputStreamWrapper> stream)
+CalculateThroughput (Ptr<OutputStreamWrapper> stream, Ptr<GamingStreamingServer> destination, uint64_t lastTotalRx)
 {
-  Time now = Simulator::Now ();    /* Return the simulator's virtual time. */
-  double cur = (packetSink->GetTotalRx () - lastTotalRx) * 8.0 / computeThroughputPeriodicity.GetSeconds () / 1e3;     /* Convert Application RX Packets to kb. */
-  averageThroughput = packetSink->GetTotalRx () * 8.0 / now.GetSeconds () / 1e6; /* Average throughput in Mb */
+  destination->GetTotalReceivedPackets ();
+  Time now = Simulator::Now ();  /* Return the simulator's virtual time. */
+  double cur = (destination->GetTotalReceivedBytes () - lastTotalRx) * 8.0 / computeThroughputPeriodicity.GetSeconds () / 1e3;  /* Convert Application RX Packets to kb */
```

Comment:
you use this formula a lot (even down below), you might want to create a function to do that, so you only have to fix it once and for all